PINCE.py

```diff
@@ -3875,6 +3880,11 @@ def copy_to_clipboard(row, column):
             current_address = utils.extract_address(current_address_text)
         menu = QMenu()
         switch_to_stacktrace = menu.addAction(tr.STACKTRACE)
+        toggle_stack_pointer = menu.addAction(tr.TOGGLE_STACK_FROM_SP_BP)
+        if debugcore.inferior_status != typedefs.INFERIOR_STATUS.RUNNING:
+            toggle_stack_pointer.setEnabled(True)
```

Comment:
There's no reason to use `setEnabled(True)`. The action starts by default with it true.

You only need to check if the inferior is running and then disable it with `setEnabled(False)`.